orchestrator/db/models.py

```diff
@@ -102,32 +103,67 @@ class ProcessTable(BaseModel):
     process_subscriptions = relationship("ProcessSubscriptionTable", lazy=True, passive_deletes=True)
     subscriptions = association_proxy("process_subscriptions", "subscription")
 
+    @property
+    @deprecated("Changed to 'process_id' from version 1.2.3, removing after version 1.3.0")
+    def pid(self) -> Column:
+        return self.process_id
+
+    @property
+    @deprecated("Changed to 'workflow_name' from version 1.2.3, removing after version 1.3.0")
+    def workflow(self) -> Column:
+        return self.workflow_name
+
+    @property
+    @deprecated("Changed to 'last_status' from version 1.2.3, removing after version 1.3.0")
+    def status(self) -> Column:
+        return self.last_status
+
+    @property
+    @deprecated("Changed to 'last_step' from version 1.2.3, removing after version 1.3.0")
+    def step(self) -> Column:
+        return self.last_step
```

Comment:
There was no `ProcessTable.status` or `ProcessTable.step` column before so not sure why we're adding them here?